src/bridge/AdminRoomHandler.ts

```diff
@@ -731,4 +731,34 @@ export class AdminRoomHandler {
         }
         return new MatrixAction("notice", null, body + "</ul>");
     }
+
+    // will mutate args if sucessful
```

Comment:
I'll allow it for convenience, but usually this is a bit scary

Reply:
Yeah, I'm not super-fond of it either, but convenience won the moment I learned that Typescript won't allow me to redefine `args` in a function like so:

```typescript
public async foo(args: string[]) {
    const [server, args] = this.extractServerFromArgs(args); // nope
    ...
}
```

I can't think of anything that'd be both nice to use like the above but not crazy action-at-a-distance like it currently is.